### PR TITLE
this is the best branch.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: test/Scrumptious.Testing.Service/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Scrumptious.Testing.Service
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
```
Comment (mrodsky) [inline test/Scrumptious.Testing.Service/Startup.cs]: Code Smell Code Smell: Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation. ([csharpsquid:S1186](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS1186))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=3&issues=AWVI3qEPiZQY_aXE0fBW&open=AWVI3qEPiZQY_aXE0fBW)

[ISSUE_KEY]: AWVI3qEPiZQY_aXE0fBW

**Review by mrodsky — COMMENTED** [review]
SonarQube analysis found issues:
Bug Bugs: 0
Vulnerability Vulnerabilities: 0
Code Smell Code Smells: 4

[See all issues in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=3&resolved=false)

File: src/Scrumptious.Service/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Scrumptious.Service
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
```
Comment (mrodsky) [inline src/Scrumptious.Service/Startup.cs]: Code Smell Code Smell: Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation. ([csharpsquid:S1186](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS1186))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=3&issues=AWVI3qEHiZQY_aXE0fBU&open=AWVI3qEHiZQY_aXE0fBU)

[ISSUE_KEY]: AWVI3qEHiZQY_aXE0fBU

File: test/Scrumptious.Testing.Service/Program.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Scrumptious.Testing.Service
+{
+    public class Program
```
Comment (mrodsky) [inline test/Scrumptious.Testing.Service/Program.cs]: Code Smell Code Smell: Add a 'protected' constructor or the 'static' keyword to the class declaration. ([csharpsquid:S1118](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=3&issues=AWVI3qENiZQY_aXE0fBV&open=AWVI3qENiZQY_aXE0fBV)

[ISSUE_KEY]: AWVI3qENiZQY_aXE0fBV

File: src/Scrumptious.Service/Program.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Scrumptious.Service
+{
+    public class Program
```
Comment (mrodsky) [inline src/Scrumptious.Service/Program.cs]: Code Smell Code Smell: Add a 'protected' constructor or the 'static' keyword to the class declaration. ([csharpsquid:S1118](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=3&issues=AWVI3qECiZQY_aXE0fBT&open=AWVI3qECiZQY_aXE0fBT)

[ISSUE_KEY]: AWVI3qECiZQY_aXE0fBT

**Review by kurtispe — APPROVED** [review]
its good for now

**Pull request merged**